zsh_completion/_kerl

```diff
@@ -16,154 +16,189 @@
 #
 # ------------------------------------------------------------------------------
 
-# TODO: These are naive and stupid
-_kerl_available_releases() {
-  # we use the file directly as `kerl list releases` dumps extraneous data to stdout
-  releases=(${(f)"$(_call_program releases cat ~/.kerl/otp_releases)"})
+_kerl_otp_releases() {
+    releases=(${(f)"$(_call_program releases cat ~/.kerl/otp_releases 2>/dev/null)"})
 }
 
-_kerl_builds() {
-  builds=(${(f)"$(_call_program builds kerl list builds 2>/dev/null | cut -f 2 -d ",")"})
```

Comment:
I prefer to rely on file contents, then have to eventually deal with side-effects these function calls might have.